src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx

```diff
@@ -21,10 +21,10 @@ import queriesnrqlTimeseriesSpikyNRQLQueryBuilder from 'images/queries-nrql_scre
 
 import queriesnrqlTimeseriesSmoothedWithSlidebyNRQLQueryBuilder from 'images/queries-nrql_screenshot-full_timeseries-smoothed-with-slide-by-NRQL-query-builder.webp'
 
-Sliding windows are a technique for generating charts using the `SLIDE BY` clause in conjunction with the `TIMESERIES` clause. With sliding windows, data is gathered in time "windows" that overlap with each other.
+If you want to generate charts using the `SLIDE BY` clause in conjunction with the `TIMESERIES` clause, you can use a technique called sliding windows. This technique enables you to gather data in time "windows" that overlap with each other.
```

Comment:
I prefer "create" to generate

Reply:
The definition feels a bit circular. Is there anyway you can define it a different way without using "windows"?